### PR TITLE
Added 'ie' class for Internet Explorer 10 and 11

### PR DESCRIPTION
**Pull request by brian-learningpool**
This is added in addition to the 'internet explorer' class.

### CODE REVIEW

File: src/core/js/device.js [hunk 1]
```diff
@@ -187,10 +187,16 @@ define(function(require) {
         }
     }
 
-    // Convert 'msie' to 'ie' for backwards compatibility
-    var browserString = (Adapt.device.browser.toLowerCase() === 'msie') ? 'ie' : Adapt.device.browser.toLowerCase();
+    var browser = Adapt.device.browser.toLowerCase();
+    // Convert 'msie' to 'ie' for backwards compatibility.
+    var browserString = (browser === 'msie') ? 'ie' : browser;
```
Comment (oliverfoster) [inline src/core/js/device.js]:
```javascript
switch(browser) {
    case "msie": case "internet explorer":
        browserString = "ie";
        break;
    case:
        browserString = browser;
}
```

or

```javascript
var browserString = browser.replace(/^msie$|^internet explorer$/gi, "ie");
```

Comment (brian-learningpool) [inline src/core/js/device.js]: Wouldn't the `switch` statement also add the `msie` class though?

Comment (oliverfoster) [inline src/core/js/device.js]:
```javascript
var browserString = browser;
switch(browserString) {
    case "msie": case "internet explorer":
        browserString = " ie";
}
```

or

```javascript
var browserString = browser.replace(/^msie$|^internet explorer$/, " ie" );
```
As per tom's latest suggestion on the original issue.

Comment (brian-learningpool) [inline src/core/js/device.js]: OK, apologies, I missed Tom's comment about disregarding the `internet explorer` class.

File: src/core/js/device.js [hunk 2]
```diff
@@ -187,8 +187,9 @@ define(function(require) {
         }
     }
 
-    // Convert 'msie' to 'ie' for backwards compatibility
-    var browserString = (Adapt.device.browser.toLowerCase() === 'msie') ? 'ie' : Adapt.device.browser.toLowerCase();
+    var browser = Adapt.device.browser.toLowerCase();
+    // Convert 'msie' and 'internet explorer' to 'ie'.
+    var browserString = browser.replace(/^msie$|^internet explorer$/, 'ie');
```
Comment (tomgreenfield) [inline src/core/js/device.js]: Could we perhaps lose the string boundary tokens? I personally find it more readable without and less need to refer back to a regex reference.
```js
var browserString = browser.replace(/msie|internet explorer/, 'ie');
```

Comment (oliverfoster) [inline src/core/js/device.js]: I'm alright with that. They aren't needed, more implied by circumstance.

**Pull request merged**